src/sass/_base.scss

```diff
@@ -115,6 +116,10 @@ html {
   @media (--phone) {
     font-size: 62.5%;
   }
+
+  @media (--large-scale) {
+    font-size: calc(62.5% + .1vw);
```

Comment:
I’ll probably keep the font-size to `62.5%`, but will think about adding a font-sizer (like on most reading-focused app, including Safari macOS reader mode and Pocket) for viewports with `width ≥ 100em`, using JavaScript.

Maybe it could be a good thing to systematically have font-size setting on a website-basis for big screen users, due to the usage (mainly: distance from the screen) being less obvious than smaller screens.